index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <meta name="description" content="Join us on the RE Global Summit 2022 at University of Toronto - Canada">
+      <link href="style.css" rel="stylesheet">
+      <title>RE Global Summit</title>
+  </head>
+  <body>
+    <header>
+      <div class="dark-container">
+        <ul class="dark-ul">
+          <li class="dark-li"><a href="#"><img src="src/social/facebook.svg" alt="facebook"></a></li>
+          <li class="dark-li"><a href="#"><img src="src/social/twitter.svg" alt="twitter"></a></li>
+          <li class="dark-li gray lato"><nav><a href="#">Logout</a></nav></li>
+          <li class="dark-li gray lato"><nav><a href="#">Mypage</a></nav></li>
+          <li class="dark-li gray lato"><nav><a href="#">English</a></nav></li>
```

Comment:
- [ ] In reference to the review comment above, I suggest that you please remove the `<nav>` tags between the `<li>` tags.